src/ngraph/frontend/onnx_import/op/abs.hpp

```diff
@@ -19,6 +19,7 @@
 #include <memory>
 
 #include "core/node.hpp"
+#include "default_opset.hpp"
```

Comment:
While *.hpp file should't contain `using namespace` we can create *.cpp file for each operator or include `ngraph/opsets/opset1.hpp` and use `ngraph::opset1` explicit instead of `default_opset.hpp`.